tests/dummy/app/templates/area-chart.hbs

```diff
@@ -0,0 +1,111 @@
+<div class="panel panel-success">
+  <div class="panel-heading">
+    <h4>Area Chart Example</h4>
+  </div>
+  <div class="panel-body">
+    <p>
+      This is a basic area chart.
+    </p>
+  </div>
+</div>
+
+<form class="form-inline">
+  <div class="form-group">
+    <label for="randomText">Text</label>
+    {{bs-input id="randomText" type="textarea" value=text placeholder="Type something in" autocomplete="off"}}
+  </div>
+  <div class="form-group">
+    <label for="heightText">Height</label>
+    {{bs-input id="heightText" type="number" min="100" max="1000" step="25" value=height}}
+  </div>
+  <div class="form-group">
+    <label for="widthText">Width</label>
+    {{bs-input id="widthText" type="number" min="100" max="1000" step="25" value=width}}
+  </div>
+</form>
+
+
+<h2>{{text}}</h2>
+
+
+<svg
+  width="{{containerWidth}}"
+  height="{{containerHeight}}"
+>
+
+  {{#with (letter-frequency text) as |letters|}}
+
+    {{#d3-graph
+      (d3-attr 'transform' (i 'translate(${x}, ${y})' x=margin.left y=margin.top))
+      as |selection|
+    }}
+
+        {{#with
+          (hash
+            xScale=(point-scale
+              (compute xDomain letters)
+              (append 0 width)
+            )
+            yScale=(linear-scale
+              (append 0 10)
+              (append height 0)
+            )
+          ) as |scales|
+        }}
+
+          {{d3-graph
+            (pipe
+              (d3-select-all 'path')
+              (d3-data (array letters))
+              (d3-join
+                enter=(pipe
+                  (d3-append 'path')
+                  (d3-attr 'class' 'area')
+                  (d3-style 'fill' 'steelblue')
+                  (d3-attr 'd'
+                    (d3-area
+                      scales.xScale
+                      scales.yScale
+                      xKey=xKey
```

Comment:
here you can use `(r/get "char")` and `(r/get "count")` once it's in ember-sparkles.